showcase/gapic-showcase/src/test/java/com/google/showcase/v1beta1/it/ITPagination.java

```diff
@@ -0,0 +1,181 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.showcase.v1beta1.it;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import com.google.api.gax.rpc.ServerStream;
+import com.google.showcase.v1beta1.EchoClient;
+import com.google.showcase.v1beta1.EchoResponse;
+import com.google.showcase.v1beta1.ExpandRequest;
+import com.google.showcase.v1beta1.PagedExpandRequest;
+import com.google.showcase.v1beta1.it.util.TestClientInitializer;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ITPagination {
+
+  private static EchoClient grpcClient;
+  private static EchoClient httpjsonClient;
+
+  @BeforeClass
+  public static void createClients() throws Exception {
+    // Create gRPC Echo Client
+    grpcClient = TestClientInitializer.createGrpcEchoClient();
+    // Create Http JSON Echo Client
+    httpjsonClient = TestClientInitializer.createHttpJsonEchoClient();
+  }
+
+  @AfterClass
+  public static void destroyClients() throws InterruptedException {
+    grpcClient.close();
+    grpcClient.awaitTermination(TestClientInitializer.AWAIT_TERMINATION_SECONDS, TimeUnit.SECONDS);
+    httpjsonClient.close();
+    httpjsonClient.awaitTermination(
+        TestClientInitializer.AWAIT_TERMINATION_SECONDS, TimeUnit.SECONDS);
+  }
+
+  @Test
+  public void testExpandGrpc() {
```

Comment:
I think we already have a [server streaming showcase test](https://github.com/googleapis/sdk-platform-java/blame/main/showcase/gapic-showcase/src/test/java/com/google/showcase/v1beta1/it/ITServerSideStreaming.java#L87-L96) for expand?

Reply:
Ah right! I forgot about that haha. 

I'll remove the expand tests and keep the pagedExpand tests in `ITPagination`.